### DISCUSSION TITLE
Add ‘target.marker.name’ and ‘target.region.name’ for Textual Feedback?

### DISCUSSION THREAD

**Discussion by PianoGeek15** [helgoboss/helgobox#1333] [Ideas]
So far I haven’t found a way to send marker or region names out the feedback output (using textual or dynamic feedback, with the mapping target “Marker/Region: Go To”). There could be and probably is a work-around way to do this, but could you make it possible to just enter ‘target.marker.name’ or ‘target.region.name’ into the textual feedback window?
Or is there another way using the region manager?

**Comment by helgoboss** [comment]
https://docs.helgoboss.org/realearn/targets/marker-region/go-to.html, see "Target-specific properties"

**Reply by PianoGeek15** [reply]
Thanks, Helgoboss! I don’t know why I didn’t find that part of the docs 😂
